src/shims/x86/mod.rs

```diff
@@ -468,6 +468,79 @@ fn unary_op_ps<'tcx>(
     Ok(())
 }
 
+enum ShiftOp {
```

Comment:
What about Srl? I suppose it doesn't make sense for some reason?

Reply:
Sla would be identical to Sll, since there isn't any difference to left-shifting signed and unsigned numbers